src/rspack/index.ts

```diff
@@ -62,6 +62,44 @@ export function getRspackPlugin<UserOptions = {}>(
           if (plugin.rspack)
             plugin.rspack(compiler)
 
+          if (plugin.watchChange || plugin.buildStart) {
+            compiler.hooks.make.tapPromise(plugin.name, async (compilation) => {
+              const context = createRspackContext(compilation)
+              if (
+                plugin.watchChange
+                && (compiler.modifiedFiles || compiler.removedFiles)
+              ) {
+                const promises: Promise<void>[] = []
+                if (compiler.modifiedFiles) {
+                  compiler.modifiedFiles.forEach(file =>
+                    promises.push(
+                      Promise.resolve(
+                        plugin.watchChange!.call(context, file, {
```

Comment:
The returning value of `watchChange` is `void`. Why do need `Promise.resolve` here?

Reply:
The `watchChange` is indeed useless, I have already removed it.